samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -33,10 +33,12 @@ static const char model_message[] =
 
 /// @brief message for performance hint
 static const char hint_message[] =
-    "Optional. Performance hint (optimize for latency or throughput)."
-    " The hint allows the OpenVINO device to select the right network-specific settings,"
-    " as opposite to just accepting specific values from the sample command line."
-    " So you can specify only the hint without setting  explicit 'nstreams' or other device-specific options";
+    "Optional. Performance hint allows the OpenVINO device to select the right network-specific settings.\n"
+    "                               'throughput' or 'tput': device performance mode will be set to THROUGHPUT.\n"
+    "                               'latency': device performance mode will be set to LATENCY.\n"
+    "                               'nan': device performance mode will be set to UNDEFINED.\n"
```

Comment:
'none' :)

Reply:
also the 'none' description should sound like **"no device performance mode will be set"**, as there is no such a real hint value.

Reply:
done. Please review